src/test/game/BoardTest.java

```diff
@@ -2,148 +2,147 @@
+        Board board = new Board();
+        List<Mark> grid = board.updateMove(1, X);
 
-        assertEquals(PLAYER_ONE, grid.get(0));
+        assertEquals(X, grid.get(0));
     }
 
     @Test
     public void canReturnRows() {
-        Board board = new Board(emptyBoard);
+        Board board = new Board();
         List<Line> rows = board.rowLines();
-        assertEquals(createNewLine("1", "2", "3"), rows.get(0));
-        assertEquals(createNewLine("4", "5", "6"), rows.get(1));
-        assertEquals(createNewLine("7", "8", "9"), rows.get(2));
+        assertEquals(createNewLine(EMPTY, EMPTY, EMPTY), rows.get(0));
+        assertEquals(createNewLine(EMPTY, EMPTY, EMPTY), rows.get(1));
+        assertEquals(createNewLine(EMPTY, EMPTY, EMPTY), rows.get(2));
     }
 
     @Test
     public void canReturnColumns() {
-        Board board = new Board(emptyBoard);
+        Board board = new Board();
         List<Line> rows = board.columnLines();
 
-        assertEquals(createNewLine("1", "4", "7"), rows.get(0));
-        assertEquals(createNewLine("2", "5", "8"), rows.get(1));
-        assertEquals(createNewLine("3", "6", "9"), rows.get(2));
+        assertEquals(createNewLine(EMPTY, EMPTY, EMPTY), rows.get(0));
+        assertEquals(createNewLine(EMPTY, EMPTY, EMPTY), rows.get(1));
+        assertEquals(createNewLine(EMPTY, EMPTY, EMPTY), rows.get(2));
     }
 
     @Test
     public void canReturnDiagonals() {
-        Board board = new Board(emptyBoard);
+        Board board = new Board();
         List<Line> diagonals = board.diagonalLines();
 
-        assertEquals(createNewLine("1", "5", "9"), diagonals.get(0));
-        assertEquals(createNewLine("3", "5", "7"), diagonals.get(1));
+        assertEquals(createNewLine(EMPTY, EMPTY, EMPTY), diagonals.get(0));
+        assertEquals(createNewLine(EMPTY, EMPTY, EMPTY), diagonals.get(1));
     }
 
     @Test
-    public void returnsListOfAvailableMoves() {
-        Board board = new Board(emptyBoard);
+    public void returnsListOfNineAvailableMoves() {
+        Board board = new Board();
 
-        assertEquals(emptyBoard, board.availableMoves());
+        assertEquals(9, board.availableMoves().size());
     }
 
     @Test
     public void returnsAListOfOneMove() {
-        Board board = new Board(asList("1", "O", "X", "O", "X", "O", "X", "O", "X"));
-        List<String> possibleMoves = asList("1");
+        Board board = new Board(asList(EMPTY, O, X, O, X, O, X, O, X));
+        List<Integer> possibleMoves = asList(0);
 
         assertEquals(possibleMoves, board.availableMoves());
     }
 
     @Test
     public void playersCantEnterSameMoveWhenMoveIsX() {
-        Board board = new Board(asList("X", "2", "3", "4", "5", "6", "7", "8", "9"));
+        Board board = new Board(asList(X, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY));
 
         assertFalse(board.isMoveAvailable(1));
     }
 
     @Test
     public void playersCantEnterSameMoveWhenMoveIsO() {
```

Comment:
I think this test and the above are the same so you can remove one of them. They're both testing the idea that you can make "a move" regardless of what the mark is.